features/standard/ContentTypeFields.feature

```diff
@@ -50,6 +50,25 @@ Feature: Content fields setting and editing
       | ezmatrix             | Matrix                       | Min_rows:2,Columns:col1-col2-col3                                     | value     | col1:col2:col3,Ala:miała:kota,Szpak:dziobał:bociana,Bociana:dziobał:szpak |            |                       |         |             | Matrix                    |
       | ezimageasset         | Image Asset                  |                                                                       | value     |  imageasset1.png.zip                                                      |            |                       |         |             | imageasset1.png           |
 
+  @javascript @common @admin
+  Scenario: Create an ImageAsset Content item and edit specified field
+    Given I create "Image" Content items in "/Media/Images/" in "eng-GB"
+      | name             | image                                                        |
+      | ImageAssetImage  | vendor/ezsystems/behatbundle/src/lib/Data/Images/small2.jpg  |
+    And I create a 'Image Asset CT2' Content Type in "Content" with 'ImageAssetCT2' identifier
```

Comment:
Nitpick: best if the `And` Steps were formatted in the same way (right now they are not aligned).